README.md

```diff
@@ -0,0 +1,205 @@
+# storybook-builder-wds
+
+[Storybook builder](https://storybook.js.org/docs/web-components/builders/overview) powered by [@web/dev-server](https://modern-web.dev/docs/dev-server/overview/).
+
+# Install
+
+Install npm packages:
+
+```bash
+npm install @web/dev-server storybook-builder-wds --save-dev
+```
+
+## Setup
+
+Follow the [official storybook docs](https://storybook.js.org/) to setup storybook.
+
+You can choose Vite when asked for the builder in the init CLI.
+Later you can change it to `storybook-builder-wds` in the storybook main configuration file.
+
+If you are migrating from the `@web/dev-server-storybook` plugin, please read the [migration guide](#migration) below.
+
+## Configuration
+
+When you finish the setup you'll have a standard storybook folder with configuration files (`.storybook` by default) containing at least `main.js`.
+
+### Activating the builder
+
+You'll need to activate `storybook-builder-wds` manually.
+To do that change the following in `main.js`:
+
+- set `config` type to `import('storybook-builder-wds').StorybookConfigWds`
+- set the `core.builder` setting to `'storybook-builder-wds'`
+- keep the `framework.name` (Vite won't be used as a builder, but the chosen framework, e.g. `web-components`, will be inherited from this setting)
+
+```js
+// .storybook/main.js
+
+/** @type { import('storybook-builder-wds').StorybookConfigWds } */
+const config = {
+  ...
+  framework: {
+    name: '@storybook/web-components-vite',
+  },
+  core: {
+    builder: 'storybook-builder-wds',
+  },
+  ...
+};
+
+export default config;
+```
+
+### Configuring storybook
+
+The `storybook-builder-wds` aims at compatibility with standard storybook features, e.g. `.storybook/preview.js`.
+Follow the [official storybook docs](https://storybook.js.org/) for the configuration options.
```

Comment:
We should add a little bit of rationale about using only browser-standard ESM, why we're using only browser-standard ESM, and that this may cause some ecosystem addons to not work if they're using non-standard stuff or cjs

Reply:
originally I was planning to, but then I found it already too long to write this initial docs and left it out for the next iteration on docs
are you OK to work on this as a separate story?